source/module_io/read_input_item_deepks.cpp

```diff
@@ -41,6 +49,31 @@ void ReadInput::item_deepks()
         item.annotation = "if set 1, prints intermediate quantities that shall "
                           "be used for making unit test";
         read_sync_bool(input.deepks_out_unittest);
+        item.reset_value = [](const Input_Item& item, Parameter& para) {
+            if (para.input.deepks_out_unittest)
+            {
+                para.input.deepks_out_labels = true;
+                para.input.deepks_scf = true;
```

Comment:
move to deepks_scf

Reply:
The parameter has been setted in the deepks_scf